libafl_concolic/test/runtime_test/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+use symcc_runtime::{
```

Comment:
this would make everything depend of SymCC (and the ability to build it, which depends on LLVM and stuff). This way, its all separate, so it can be tested and built only on targets that actually support it

Reply:
Feature flag?

Reply:
also put this on the todo list :)

Reply:
Okay, so I checked this again. The problem is that runtime_test needs to be a separate crate, because it needs to produce the separate shared object file as compilation result. therefore, I think I can't put it in the `libafl_test` crate